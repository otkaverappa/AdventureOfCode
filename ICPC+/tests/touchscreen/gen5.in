19
hqddj 10
ipftt
bosoa
otnbz
qdwgq
cqggz
fnqit
btkim
dlcvb
dmusn
uksqr
kuhra 10
gyqyi
jcbtk
owwst
xvhup
uoaog
rzbas
bgsth
aejcx
vsvtk
rriyn
xvbxj 10
hqkkq
eozwh
gxmpb
lmtjg
dazla
mjvni
hwzth
pyxqu
gwtsn
uganr
gttfe 10
ttprg
ayebt
nsrkj
orfjl
vetvt
kdmfk
tzdks
lmspn
lfgdp
ptiwc
wrgro 10
ccrqj
djigw
arisj
wgocj
fteqp
gmhpd
vrhph
rstbz
peszy
bverz
pzstp 10
iadrp
ioisd
pjykm
zbstb
rxyxo
ynnsg
ecihv
zqkhi
pztpj
holai
otfmq 10
vmdlh
mpjwx
fwprf
xiesy
qzmdb
wrudf
lzsqk
zczkb
yrxni
cmrje
pbgde 10
jcxef
epewh
qxksk
lqbkg
mosfz
zwahz
gqbfw
jklpi
tgifa
sssve
yhuto 10
uukwb
mftpk
qywdo
hywpf
wjzoe
dpoyi
cucnq
fbvyq
iqpeu
fnubc
aznzn 10
sfcid
mkzoz
pvanw
rxmid
inrdq
tfphe
cblhj
ovvpm
xfhzu
dsrsa
uaplf 10
fhkuq
rzrci
ctdzi
pypzy
jcqcu
txxkk
cqrpm
hglak
vedze
oqcdp
cpuvr 10
oqqla
bqssf
fcnse
xojdn
ptftw
vwnpt
ffjvt
jwjer
qjvdb
zcpih
examt 10
zhrmy
ktftr
aenji
gbtbg
vblml
srlvd
eumvg
mhctc
tuiif
tphms
pjvaw 10
hvnsq
sykgv
sucwn
ephmy
pfpyu
hnddq
zmlog
bieno
bhidg
yjvhx
vwdku 10
zskcx
acjls
pobvd
rwmcb
sakqj
hlgmw
clqmq
pouyz
mppnm
uhkjj
lblxt 10
ugfav
dfitt
yiisi
jhxzu
lvbve
mgixf
bsliu
glzpe
vpodk
wmrvn
ngjrb 10
pdjxb
obvcj
pkvrb
bmrqp
boduk
rhscy
wrcfr
fwsay
briyi
mawdq
lgera 10
qihjm
ifdkm
wrior
isktq
tfqpk
idqnw
sdgbo
sjtxv
gumqk
fzcqu
vlzlc 10
juhbj
dtmju
bbewb
bevov
fvwkn
qfypq
damkb
vowkx
qnbwj
cyqzo
