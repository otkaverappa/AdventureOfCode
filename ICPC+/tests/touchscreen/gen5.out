cqggz 15
btkim 18
dlcvb 18
uksqr 18
dmusn 19
ipftt 19
qdwgq 20
fnqit 21
otnbz 22
bosoa 24
aejcx 17
bgsth 17
jcbtk 17
uoaog 18
rriyn 19
gyqyi 20
owwst 20
rzbas 20
vsvtk 22
xvhup 25
gwtsn 13
mjvni 16
eozwh 17
hwzth 19
gxmpb 21
lmtjg 21
pyxqu 21
uganr 22
dazla 24
hqkkq 28
vetvt 7
ttprg 10
ayebt 11
ptiwc 14
lfgdp 15
kdmfk 16
nsrkj 16
tzdks 16
orfjl 18
lmspn 24
fteqp 11
arisj 12
wgocj 13
ccrqj 14
vrhph 15
djigw 19
rstbz 19
bverz 20
peszy 20
gmhpd 23
iadrp 5
pztpj 13
rxyxo 18
pjykm 21
zbstb 22
zqkhi 23
ynnsg 24
ecihv 26
holai 29
ioisd 31
yrxni 15
cmrje 16
vmdlh 21
wrudf 21
mpjwx 22
fwprf 25
lzsqk 25
xiesy 25
zczkb 26
qzmdb 29
jcxef 13
tgifa 14
gqbfw 15
lqbkg 17
sssve 18
mosfz 19
qxksk 22
epewh 23
jklpi 25
zwahz 27
dpoyi 13
fnubc 14
mftpk 14
qywdo 14
uukwb 14
iqpeu 15
hywpf 18
fbvyq 19
wjzoe 23
cucnq 24
xfhzu 10
dsrsa 16
cblhj 19
rxmid 19
sfcid 21
ovvpm 26
tfphe 26
inrdq 27
pvanw 30
mkzoz 35
cqrpm 19
fhkuq 19
txxkk 19
hglak 20
pypzy 21
rzrci 22
jcqcu 24
oqcdp 25
vedze 27
ctdzi 33
xojdn 9
ffjvt 11
jwjer 17
fcnse 18
qjvdb 18
bqssf 21
vwnpt 21
ptftw 23
zcpih 23
oqqla 33
gbtbg 14
zhrmy 14
aenji 16
eumvg 18
ktftr 19
vblml 19
srlvd 20
tphms 21
mhctc 22
tuiif 22
yjvhx 11
hnddq 12
hvnsq 13
sucwn 19
bhidg 20
pfpyu 22
sykgv 24
bieno 25
ephmy 25
zmlog 31
rwmcb 17
uhkjj 17
acjls 18
sakqj 19
clqmq 20
hlgmw 20
zskcx 22
mppnm 24
pobvd 28
pouyz 31
mgixf 8
lvbve 10
ugfav 14
dfitt 15
jhxzu 15
yiisi 15
bsliu 19
vpodk 20
wmrvn 21
glzpe 30
briyi 12
pdjxb 12
pkvrb 13
obvcj 16
rhscy 16
boduk 17
wrcfr 17
bmrqp 18
fwsay 19
mawdq 19
idqnw 12
isktq 13
ifdkm 16
sjtxv 19
tfqpk 21
fzcqu 22
gumqk 23
sdgbo 24
wrior 24
qihjm 28
vowkx 6
bevov 13
fvwkn 14
damkb 19
bbewb 20
dtmju 21
juhbj 23
qfypq 23
cyqzo 24
qnbwj 26
